pkgs/top-level/python-packages.nix

```diff
@@ -1181,6 +1181,8 @@ in {
 
   backports_abc = callPackage ../development/python-modules/backports_abc { };
 
+  backports_cached-property = callPackage ../development/python-modules/backports_cached-property { };
```

Comment:
```suggestion
  backports-cached-property = callPackage ../development/python-modules/backports-cached-property { };
```

Reply:
I chose package name like in these packages - `backports_csv`, `backports_abc`, `backports_unittest-mock`, `backports-shutil-which` and etc.

Reply:
Those names are all not following the documentation and should be updated https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md?plain=1#L1718-L1723

Reply:
Renamed package.